README.md

```diff
@@ -1,14 +1,16 @@
 # Cloud Code for Visual Studio Code
 
-Cloud Code for VS Code brings the power and convenience of IDEs to cloud-native Kubernetes and Cloud Run application development. Cloud Code works with Google’s command-line [container tools](https://github.com/GoogleContainerTools) like [`skaffold`](https://skaffold.dev/) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) under the hood, providing local, continuous feedback on your project as you build, edit, and run your applications locally or in the cloud.
+Cloud Code for VS Code brings the power and convenience of IDEs to cloud-native Kubernetes and Cloud Run application development. Cloud Code integrates with a chain of Google Cloud products(like GKE, Cloud Run, Cloud APIs, Secret Manager) and makes you feel like you are working with local code. 
```

Comment:
Paring down the wording a bit:

Cloud Code for VS Code brings the power and convenience of IDEs to cloud-native application development. With its integration of Google Cloud services like Google Kubernetes Engine, Cloud Run, Cloud APIs, and Secret Manager, Cloud Code makes you feel like you're working with local code.

Reply:
Updated.